presto-native-execution/presto_cpp/main/connectors/IcebergPrestoToVeloxConnector.cpp

```diff
@@ -42,6 +43,83 @@ velox::dwio::common::FileFormat toVeloxFileFormat(
   VELOX_UNSUPPORTED("Unsupported file format: {}", fmt::underlying(format));
 }
 
+std::unique_ptr<velox::connector::ConnectorTableHandle> toIcebergTableHandle(
+    const protocol::TupleDomain<protocol::Subfield>& domainPredicate,
```

Comment:
In general I would expect the first 3 parameters to come to the end of the function signature and tableName, dataColumns, tableHandle etc be the beginning parameters. Can that be changed ?